### PR TITLE
add WIP "how we work" document for Backend area

### PR DESCRIPTION
**Pull request by hotzevzl**

### CODE REVIEW

File: _docs/backenders/how-we-work/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+to plan ahead, discuss functional design, unblock other project members' work.
+
+We may occasionally pair up remotely to work together on specific functional
+design tasks, to develop features, triage and fix bugs and to seek feedback and
+advice, although we don't use pair programming as a daily way of working.
+
+We take code reviews both as an opportunity to improve our projects and to learn
+and mentor.
+
+We share interesting bits of our daily work at our weekly team meeting. We do
```
Comment (agnessa) [inline _docs/backenders/how-we-work/index.md:49]: Let's rewrite this to reflect two types of meetings. e.g. We meet twice a week for two distinct types of team meetings. The first one is an organisational one, dedicated to updating each other and making team decisions broadly related to running a self-managed organisation. The other one is a technical meeting, where we share interesting bits of our daily work. We do also share frustrations, celebrate achievements and in general to sustain the team's growth, while keeping a light-hearted spirit. Once a month that meeting is held together with the Front End engineers team.

**Review by agnessa — CHANGES_REQUESTED** [review]

File: _docs/backenders/how-we-work/index.md (hunk 1 above)
Comment (hotzevzl) [reply _docs/backenders/how-we-work/index.md]: great, thanks for the text suggestion - I have used it verbatim ❤️

**Review by tiagojsag — COMMENTED** [review]
two things:
- Elasticsearch is not a database (the fact we're using it as one is why I find it so evil)
- I would rephrase the "we use docker" to "we may use docker". I dont use it myself

**Review by alexeh — APPROVED** [review]

**Review by agnessa — APPROVED** [review]
